pepper-apis/dsm-core/src/main/java/org/broadinstitute/lddp/util/JsonTransformer.java

```diff
@@ -1,11 +1,12 @@
 package org.broadinstitute.lddp.util;
 
 import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 import spark.ResponseTransformer;
 
 public class JsonTransformer implements ResponseTransformer {
 
-    private Gson gson = new Gson();
+    private Gson gson = new GsonBuilder().serializeNulls().create();
```

Comment:
My first lines of thought was, we should return all of the variables even if they are null, because otherwise it will look like the values for them are just not provided in the response. But now I agree with you about the impact of this file, so I thought with Juniper folks and Brian specifically and we came to the decision that Juniper will assume anything that is not present in the response was null